FEXCore/Source/Interface/Core/JIT/Arm64/BranchOps.cpp

```diff
@@ -427,10 +427,11 @@ DEF_OP(CPUID) {
   PopDynamicRegsAndLR();
 
   // Results are in x0, x1
-  // Results want to be in a i64v2 vector
-  auto Dst = GetRegPair(Node);
-  mov(ARMEmitter::Size::i64Bit, Dst.first, TMP1);
-  mov(ARMEmitter::Size::i64Bit, Dst.second, TMP2);
+  // Results want to be 4xi32 scalars
+  mov(ARMEmitter::Size::i32Bit, GetReg(Op->OutEAX.ID()), TMP1);
+  mov(ARMEmitter::Size::i32Bit, GetReg(Op->OutECX.ID()), TMP2);
+  lsr(ARMEmitter::Size::i64Bit, GetReg(Op->OutEBX.ID()), TMP1, 32);
```

Comment:
You'll want to change these `lsr` instructions in CPUID and XGETBV to `bfe` instructions instead. lsr has lower throughput than bfe.

Reply:
Done. Performance should be the same on firestorm, I'm guessing this is a cortex quirk?

Reply:
Indeed. lsr > 4 goes down the M pipeline, bfe goes down all the integer pipelines.

Reply:
Wacky wacky. Anyway, apparently I wasn't pushing the right branch :laughing: should be there now